modules/compose/compose_api_test.go

```diff
@@ -365,6 +367,29 @@ func TestDockerComposeAPIWithVolume(t *testing.T) {
 	assert.NoError(t, err, "compose.Up()")
 }
 
+func TestDockerComposeAPIVolumesDeletedOnDown(t *testing.T) {
+	identifier := uuid.New().String()
+	stackFiles := WithStackFiles("./testresources/docker-compose-volume.yml")
```

Comment:
Could you use filepath to create a OS-agnostic path? 🙏

Reply:
I know it's not there for any method, if you want to add them all, in a separate PR, that would be awesome

Reply:
Updated all tests to use `filepath` 👍

Reply:
This is a tiny thing, but I'd appreciate if you extract the file names to a constant, as Sonar is "complaining" about it: https://sonarcloud.io/project/issues?id=testcontainers_testcontainers-go&pullRequest=910&resolved=false&types=CODE_SMELL 🙏

Reply:
@cgoodsirsmyth-pp I this is the only thing left 🙏  As I said, it's a tiny one, but if you think we could live with the sonar warning which, to me, in testing sometimes is desired to have repetition, then it's fine to merge it as is.

Reply:
Moved simple, complex and volume compose to a `const` as well as the `testresources` package.

Reply:
Did not notice the sonarcloud code smell apologies.

Reply:
Ah no worries, we are experimenting with sonarcloud atm. Not very strict with them, but checking if it provides value for us. Thanks anyway for updating the PR. I think we are ready to go now :)